client/app/views/receiver/tip.html

```diff
@@ -79,19 +79,65 @@
   <div class="row">
     <div class="col-md-6">
       <div class="TipPageSectionTitle" data-translate>Submission</div>
-      <div class="col-md-12">
-        <dl>
-          <dt>
-            <span>{{::tip.context_name}}</span>
+
+      <div class="SubmissionInfoSection">
+      <table>
+
+        <tbody>
+        <tr>
+          <td class="left">
+            <span data-translate>Context</span>
+          </td>
+          <td class="right">
+            <span><b>{{::tip.context_name}}</b></span>
             <a data-ng-show="tip.context_description" popover="{{::tip.context_description}}" popover-placement="right" popover-trigger="mouseenter">
               <span class="glyphicon glyphicon-info-sign"></span>
             </a>
-          </dt>
-          <dt><span data-translate>Submission date</span></dt>
-          <dd>{{::tip.creation_date | date:'dd-MM-yyyy HH:mm'}}</dd>
-          <dt><span data-translate>Expiration Date</span></dt>
-          <dd>{{::tip.expiration_date | date:'dd-MM-yyyy HH:mm'}}</dd>
-        </dl>
+          </td>
+        </tr>
+        <tr>
+          <td class="left">
+            <span data-translate>Submission date</span>
+          </td>
+          <td class="right">
+            {{::tip.creation_date | date:'dd-MM-yyyy HH:mm'}}
+          </td>
+        </tr>
+        <tr>
+          <td class="left">
+            <span data-translate>Expiration Date</span>
+          </td>
+          <td class="right">
+            {{::tip.expiration_date | date:'dd-MM-yyyy HH:mm'}}
+          </td>
+        </tr>
+        <tr>
+          <td class="left">
+            <span data-translate>Whistleblower security</span>
+          </td>
```

Comment:
I'd use a a non-textual representation with an icon somewhere with a icon representing a "Connection" that can then have a value: HTTPS / Tor .
Then a hint with on-mouse-over that will say:
HTTPS: "The connection to send this submission was encrypted using HTTPS, ensuring confidentiality"
Tor: "The connection to send this submission was encrypted using Tor, ensuring confidentiality and anonymity" 

Then this indicator must be configurable on/off (because it's a TECHNICAL indicator, that's not likely in the interests of any lawyer handling anticorruption submissions).